internal/pkg/template/templates/addons/ddb/env/cf.yml

```diff
@@ -0,0 +1,48 @@
+Parameters:
+  App:
+    Type: String
+    Description: Your application's name.
+  Env:
+    Type: String
+    Description: The name of the environment being deployed.
+
+Resources:
+  {{logicalIDSafe .Name}}:
+    Metadata:
+      'aws:copilot:description': 'An Amazon DynamoDB table for {{.Name}}'
+    Type: AWS::DynamoDB::Table
+    Properties:
+      TableName: !Sub ${App}-${Env}-{{.Name}}
+      AttributeDefinitions:{{range .Attributes}}
+        - AttributeName: {{.Name}}
+          AttributeType: "{{.DataType}}"{{end}}
+      BillingMode: PAY_PER_REQUEST
+      KeySchema:
+        - AttributeName: {{.PartitionKey}}
+          KeyType: HASH{{ if .SortKey }}
+        - AttributeName: {{.SortKey}}
+          KeyType: RANGE{{end}}{{if .HasLSI}}
+      LocalSecondaryIndexes:{{range .LSIs}}
+        - IndexName: {{.Name}}
+          KeySchema:
+            - AttributeName: {{.PartitionKey}}
+              KeyType: HASH
+            - AttributeName: {{.SortKey}}
+              KeyType: RANGE
+          Projection:
+            ProjectionType: ALL{{end}}{{end}}
+
+Outputs:
+  {{envVarName .Name}}:
+    Description: "The name of this DynamoDB table."
+    Value: !Ref {{logicalIDSafe .Name}}
+#    # Uncomment the "Export" block if you need to inject the table name as an environment variable in a workload. 
+#    Export: 
+#      Name: !Sub ${Env}-{{logicalIDSafe .Name}}TableName
+  {{logicalIDSafe .Name}}DynamoDBTableARN:
+    Description: "The ARN of the {{.Name}} DynamoDB table."
+    Value: !GetAtt {{logicalIDSafe .Name}}.Arn
+#    # Uncomment the "Export" block if you need to reference the DynamoDB table ARN in a workload addon.
+#    Export: 
+#      Name: !Sub ${Env}-{{logicalIDSafe .Name}}ARN
```

Comment:
nit: I wonder if for env addons we should just leave this uncommented

Reply:
Good point! 

There are two benefits that I had in mind for having this commented.

The first is to familiarize users with the "export + `from_cfn`" procedure. Instead of having the template connectable to workloads out-of-the-box, this provides a hands-on guidance. 

The second benefit is that, this (hopefully) incentivizes users to look at what an `Export` is, and signals that `Export` is only needed if the value will be used in a workload. Since this also serves as an example of how users could write an environment addon, I'm worried that users would just `Export` every single `Output` in any env addons that they write. 

The benefit of leaving this uncommented is obvious - it saves users some extra steps 👍🏼  I think maybe we can take a middle ground here:
```yaml
  {{envVarName .Name}}:
    Description: "The name of this DynamoDB table."
    Value: !Ref {{logicalIDSafe .Name}}
    # NOTE: 
    # You can inject the table name as an environment variable into a workload:
    #  ```
    # variables:
    #    {{logicalIDSafe .Name}}_TABLE_NAME: 
    #        from_cfn: ${COPILOT_ENVIRONMENT_NAME}-{{logicalIDSafe .Name}}TableName
    # ```
    # Remove the "Export" if none of your workload needs this as an environment variable.
    Export:    
      Name: !Sub ${Env}-{{logicalIDSafe .Name}}TableName
  {{logicalIDSafe .Name}}DynamoDBTableARN:
    Description: "The ARN of the {{.Name}} DynamoDB table."
    Value: !GetAtt {{logicalIDSafe .Name}}.Arn
    # NOTE: ...similar but with an example to workload addon
    Export: 
      Name: !Sub ${Env}-{{logicalIDSafe .Name}}ARN
```